controllers/controller.go

```diff
@@ -159,7 +158,6 @@ func (s *GetItemDockerControllerImpl) GetDockerConfig() http.HandlerFunc {
 				http.Error(w, "Record not found of this ID:"+vars["id"], http.StatusNotFound)
 				return
 			} else {
-				home := os.Getenv("HOME")
 				tmpl, err := template.ParseFiles(home + "/view/DockerData.html")
 				if err != nil {
 					fmt.Println(errors.New("unable to execute the template"))
```

Comment:
return http message with error instead of prinitng the error.

Reply:
fixed